android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -1154,21 +1164,45 @@ public static void setUpTransactionList(BraveWalletBaseActivity activity,
             pendingTxHelper.fetchTransactions(() -> {
                 HashMap<String, TransactionInfo[]> pendingTxInfos =
                         pendingTxHelper.getTransactions();
-                workWithTransactions(activity, selectedNetwork, pendingTxInfos, accounts,
-                        walletListItemModel, assetPrices, fullTokenList, nativeAssetsBalances,
-                        blockchainTokensBalances, rvTransactions, callback, walletTxCoinAdapter);
+                SolanaTransactionsGasHelper solanaTransactionsGasHelper =
```

Comment:
Also this can be just a static function, like what I did with `BalanceHelper`. But I can do that later if that's too much things in this PR.

Reply:
yeah, it could be, let's do it later.